src/sentry/event_manager.py

```diff
@@ -477,6 +479,17 @@ def to_values(v):
             exception['values'][0]['stacktrace'] = stacktrace
             del data['sentry.interfaces.Stacktrace']
 
+        # Exception mechanism needs SDK information to resolve proper names in
+        # exception meta (such as signal names). "SDK Information" really means
+        # the operating system version the event was generated on. Some
+        # normalization still works without sdk_info, such as mach_exception
+        # names (they can only occur on macOS).
+        if exception:
+            sdk_info = get_sdk_from_event(data)
+            for ex in exception['values']:
+                if 'mechanism' in ex:
+                    normalize_mechanism_meta(ex['mechanism'], sdk_info)
```

Comment:
We need `debug_meta.sdk_info` to normalize mechanism meta. This is why we have to do this in EventManager as opposed to the Mechanism interface.